platform/openide.explorer/src/org/netbeans/modules/openide/explorer/ExternalDragAndDrop.java

```diff
@@ -59,7 +59,7 @@ public static Transferable maybeAddExternalFileDnd( Multi multi ) {
         Transferable res = multi;
         try {
             MultiTransferObject mto = (MultiTransferObject) multi.getTransferData(ExTransferable.multiFlavor);
-            final ArrayList fileList = new ArrayList( mto.getCount() );
+            final List fileList = new ArrayList<>( mto.getCount() );
```

Comment:
List<Object> for the left hand side.

Reply:
Hey @mbien , I missed this by accident.. I will get it it in though with another change.. Thanks for the feedback..

Reply:
no worries @BradWalker keep up the great (renovation) work. There is always the next PR :)

Reply:
btw @BradWalker i have some code cleanup in this PR https://github.com/apache/netbeans/pull/3171 (java.hints.declarative module). Maybe we can coordinate to not end up having merge conflicts.

Reply:
@mbien , absolutely.. just tag me when you have a code review ready and I'll look at everything.. It would help me to learn..